### PR TITLE
Added Installation instructions to README

### PR DESCRIPTION
**Pull request by valllllll2000**

### CODE REVIEW

File: Android Workshop/README.md [hunk 1]
```diff
@@ -1,2 +1,132 @@
...
+## Installation instructions
+ 
+### Step 1: Install Java
+     
+#### Java installation verification
+You will need to open a Terminal.
+  - Windows: go to the search (at the bottom left in Windows 10) and type `cmd`. 
+You will see a program in the list, select it.
+  - Linux: ctr + alt + T or look for the app Terminal
+  - Mac: look for the app Terminal (often inside tools??)
```
Comment (Perrankana) [inline Android Workshop/README.md]: I have it in "others" folder

Comment (coffeearmy) [inline Android Workshop/README.md]: is in other ( at least in my mac) But don't put anything, they should know where is it. And i'm not sure that the folder changes from different versions

Comment (valllllll2000) [reply Android Workshop/README.md]: I have removed that instruction

Comment (valllllll2000) [reply Android Workshop/README.md]: I have removed it

**Comment by valllllll2000** [comment]
@coffeearmy I though the same as you but in the codelab https://codelabs.developers.google.com/codelabs/build-your-first-android-app/#1
you can see they say 

> Important: Don't install Android Studio until after the Java JDK is installed. Without a working copy of Java, the rest of the process will not work
So that is why I added the instructions.

**Pull request merged**